File: TicketReservation/src/main/java/pl/ticketreservation/screening/ScreeningDTO.java
package pl.ticketreservation.screening;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import pl.ticketreservation.movie.Movie;
import pl.ticketreservation.seat.Seat;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ScreeningDTO {
    private Screening screening;
    private Movie movie;
    private List<Seat> seats;
    private List<Seat> availableSeats;
}

File: ticket-reservation-app/src/app/cinema/models/screening-dto.ts
import {Screening} from "./screening";
import {Movie} from "./movie";
import {Seat} from "./seat";

export class ScreeningDto {
  screening: Screening = new Screening()
  movie: Movie = new Movie();
  seats: Seat[] = new Array<Seat>
  availableSeats: Seat[] = new Array<Seat>
  constructor() {
  }
}
